### PR TITLE
downgrade node to version 18.18.2

### PR DESCRIPTION
**Pull request by MrsTonedOne**
it works on windows7 , as long as you don't attempt to resolve a dns address on the node/server side. which we don't

@jlsnow301 will likely make a pr to duel test 18 and 20 and close this one, but its here if they don't.

### CODE REVIEW

File: dependencies.sh [hunk 1]
```diff
@@ -11,8 +11,9 @@ export BYOND_MINOR=1633
 export RUST_G_VERSION=3.1.0
 
 #node version
-export NODE_VERSION=20
-export NODE_VERSION_LTS=20.12.0
+NODE_SKIP_PLATFORM_CHECK=1
+export NODE_VERSION=18
```
Comment (optimumtact) [inline dependencies.sh:15]:
```
# IF YOU CHANGE THIS MAKE SURE IT BUILDS ON WINDOWS 7 OR ORANGES WILL PUNCH A HOLE STRAIGHT THROUGH YOUR HEART
```

**Comment by jlsnow301** [comment]
why 18.18.2 in particular? you mentioned that we should use v18 because this was LTS, but this is not the LTS as of writing

**Comment by MrStonedOne** [comment]
> why 18.18.2 in particular? 

the stack overflow thread said it worked and i misread 18.19 as 18.18 so i thought i was already testing the highest 18 version.

18.19 works i just checked but i think it will likely make you the happiest to re-setup two 20 tracks and have CI test for them both.

just to have it noted somewhere: in that situation 20.2.0 would be the compat version

File: dependencies.sh (hunk 1 above)
Comment (MrStonedOne) [reply dependencies.sh:15]:
```suggestion
# IF YOU CHANGE THIS MAKE SURE IT BUILDS ON WINDOWS 7 OR ORANGES WILL PUNCH A HOLE STRAIGHT THROUGH YOUR HEART
export NODE_VERSION=18
```

File: dependencies.sh [hunk 2]
```diff
@@ -11,8 +11,10 @@ export BYOND_MINOR=1633
 export RUST_G_VERSION=3.1.0
 
 #node version
-export NODE_VERSION=20
-export NODE_VERSION_LTS=20.12.0
+NODE_SKIP_PLATFORM_CHECK=1
+# IF YOU CHANGE THIS MAKE SURE IT BUILDS ON WINDOWS 7 OR ORANGES WILL PUNCH A HOLE STRAIGHT THROUGH YOUR HEART
+export NODE_VERSION=18
+export NODE_VERSION_LTS=18.18.2
```
Comment (MrStonedOne) [inline dependencies.sh:17]:
```suggestion
export NODE_VERSION_LTS=18.19.1
```

**Comment by jlsnow301** [comment]
> 18.19 works i just checked

do you mind linking the article? I spun up a win 7 vm yesterday and couldn't get it working (I also maybe tried 18.18). Curious to see what sorcery it's using

**Pull request closed without merging**